### PR TITLE
macos: update installer background

### PR DESCRIPTION
**Pull request by surendrajat**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [ ] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

Package installer does not set bg logo properly and it looks bad:

Screenshot 2021-11-18 at 10 21 50 PM

This PR creates an extended 620x418 (installer size) image bg on-the-fly with `imagemagick` during pkg build. It looks like this now:

Screenshot 2021-11-19 at 12 24 11 PM

I've also updated the current `rizin-logo.png`.

**Test plan**

See if the CI artifact installer has correct bg.

**Closing issues**

...

### CODE REVIEW

File: dist/osx/build_osx_package.sh [hunk 1]
```diff
@@ -22,6 +22,9 @@ cd "${RIZININSTALL}" && find . | cpio -o --format odc | gzip -c > "${OSXPKGDIR}/
 mkbom "${RIZININSTALL}" "${OSXPKGDIR}/Bom"
 pkgutil --flatten "${OSXPKGDIR}" "${RIZINDIR}/dist/osx/rizin.pkg"
 
+# convert -size 620x418 xc:none -background none /tmp/bg.png
+# composite -geometry +30+230 -background none \( "${RIZINDIR}/doc/img/rizin.svg" -resize 25% \) /tmp/bg.png  -colorspace sRGB "${RIZINDIR}/dist/osx/Resources/rizin-logo.png"
```
Comment (thestr4ng3r) [inline dist/osx/build_osx_package.sh]: Any reason to have it in the yml and not here? I think it fits better into this script to have everything self-contained.

Comment (surendrajat) [reply dist/osx/build_osx_package.sh]: It'll introduce dependency on `imagemagick`. That needs to be documented as well, but I couldn't find a good place.

Comment (thestr4ng3r) [reply dist/osx/build_osx_package.sh]: You could do something like `convert ... || { echo convert failed, is imagemagick installed?; exit 1; }`
Then one will see right away what is wrong.

Comment (surendrajat) [reply dist/osx/build_osx_package.sh]: We can probably remove the current `rizin-logo.png` completely now?

**Comment by ret2libc** [comment]
I think you should install ImageMagick in the CI jobs then.

**Comment by surendrajat** [comment]
> I think you should install ImageMagick in the CI jobs then.

Removed by mistake 😬

File: dist/osx/build_osx_package.sh (hunk 1 above)
Comment (thestr4ng3r) [reply dist/osx/build_osx_package.sh]: Yes (as you have already done)

**Review by thestr4ng3r — APPROVED** [review]

**Pull request merged**